db/migrate/20151110181710_change_collation_for_tag_names.acts_as_taggable_on_engine.rb

```diff
@@ -0,0 +1,10 @@
+# This migration comes from acts_as_taggable_on_engine (originally 5)
+# This migration is added to circumvent issue #623 and have special characters
+# work properly
+class ChangeCollationForTagNames < ActiveRecord::Migration
+  def up
+    if ActsAsTaggableOn::Utils.using_mysql?
```

Comment:
we do have setu specific to postgres in in `config/databse.yml` but keeping this wont hurt, not a big deal